index.html

```diff
@@ -56,6 +56,13 @@ <h1 class="logo">Coders<br />who<br />Teach</h1>
       </div>
     </header>
     <main>
+      <div class="alert">
+        <h3>Come to our November re-launch!</h3>
```

Comment:
It would be nice to keep the heading structure logical -- making this a `h2` (and extending the `.alert` heading CSS to `h2`s) or adding a visibly hidden `h2` "News" or similar (CSS in the [`.visibly-hidden` code snippet](https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/)).
This is not a blocker though as there isn't too much content and there are only two headings :) so your call!

Reply:
Good note, thanks! I've used your first suggestion.